x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/test/TestRestrictedIndices.java

```diff
@@ -8,37 +8,54 @@
 
 package org.elasticsearch.xpack.core.security.test;
 
-import org.apache.lucene.util.automaton.Automaton;
 import org.elasticsearch.Version;
 import org.elasticsearch.cluster.metadata.IndexNameExpressionResolver;
 import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.common.util.set.Sets;
 import org.elasticsearch.indices.ExecutorNames;
 import org.elasticsearch.indices.SystemIndexDescriptor;
 import org.elasticsearch.indices.SystemIndices;
 import org.elasticsearch.indices.SystemIndices.Feature;
 import org.elasticsearch.indices.TestIndexNameExpressionResolver;
 import org.elasticsearch.xcontent.XContentBuilder;
 import org.elasticsearch.xpack.core.XPackPlugin;
-import org.elasticsearch.xpack.core.security.index.RestrictedIndicesNames;
+import org.elasticsearch.xpack.core.security.authz.RestrictedIndices;
 
 import java.io.IOException;
 import java.io.UncheckedIOException;
+import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 
 import static org.elasticsearch.index.mapper.MapperService.SINGLE_MAPPING_NAME;
 import static org.elasticsearch.xcontent.XContentFactory.jsonBuilder;
 import static org.elasticsearch.xpack.core.ClientHelper.ASYNC_SEARCH_ORIGIN;
 import static org.elasticsearch.xpack.core.ClientHelper.SECURITY_ORIGIN;
-import static org.elasticsearch.xpack.core.security.index.RestrictedIndicesNames.SECURITY_MAIN_ALIAS;
-import static org.elasticsearch.xpack.core.security.index.RestrictedIndicesNames.SECURITY_TOKENS_ALIAS;
 
 public class TestRestrictedIndices {
 
-    public static final Automaton RESTRICTED_INDICES_AUTOMATON;
+    public static final RestrictedIndices RESTRICTED_INDICES;
     public static final IndexNameExpressionResolver RESOLVER;
 
+    public static final String SECURITY_MAIN_ALIAS = ".security";
+    public static final String INTERNAL_SECURITY_MAIN_INDEX_7 = ".security-7";
+    public static final String INTERNAL_SECURITY_MAIN_INDEX_6 = ".security-6";
+
+    public static final String SECURITY_TOKENS_ALIAS = ".security-tokens";
+    public static final String INTERNAL_SECURITY_TOKENS_INDEX_7 = ".security-tokens-7";
+
+    public static final Set<String> RESTRICTED_NAMES = Collections.unmodifiableSet(
```

Comment:
As long as we're touching basically every line this is used on anyway, it might be a good opportunity to rename this to something like `SAMPLE_RESTRICTED_NAMES` or something, because it's no longer a canonical list, just a list of names we test.